test/run-integration-tests.yml

```diff
@@ -16,6 +17,12 @@ steps:
     DOTNET_SDK_VERSION: ${{ parameters.dotnetSdkVersion }}
     PROJECTS: ${{ parameters.projectName }}
   displayName: Check for required parameters in YAML template
+- powershell: /
```

Comment:
Just curious, how is `/` different from  `|`?

Reply:
Ah, not sure; I thought that `/` was for multi-line.

Reply:
No, that would be `|`.